bloom-housing/bloom
# Bug/1843/application types

## Issue

Addresses #1843 


## Description

This fixes Issues with the Apply section and different application types. This includes updates to the ApplicationTypes section. This also renames `openDateState` to `openInFuture`, which is more descriptive and pulls logic out of UI-Components and is a breaking change. This also has a fix for the re-submit issues on the listing form.

## Type of change


## How Can This Be Tested/Reviewed?

You should be able to to add digital, paper and referral types and have them all visible on the public site as expected. This is a change in logic, where before if there was a referral type, the digital and paper applications won't show. Since a user in the partner's listing management form can now add each type, it makes sense to show all the types the user has specified.


## Checklist:
